apps/meteor/client/components/GenericNoResults/GenericNoResults.tsx

```diff
@@ -9,22 +9,40 @@ type GenericNoResultsProps = {
 	description?: string;
 	buttonTitle?: string;
 	buttonAction?: () => void;
```

Comment:
Could we use a union instead of optionals here? E.g. it would prevent passing `buttonTitle` without `buttonAction`.

Reply:
```suggestion
} & ({ 
	buttonTitle: string;
	buttonAction: () => void;
} | { 
	buttonTitle?: never;
	buttonAction?: never;
}) & {
```